integration/setup_test.go

```diff
@@ -31,22 +31,24 @@ func (s *integrationTestSuite) Setup() {
 		o := obj
 		// check if object already exists
 		var existingObj client.Object
-		_ = integration.Client.Get(ctx, client.ObjectKey{
+		err := integration.Client.Get(ctx, client.ObjectKey{
 			Namespace: o.GetNamespace(),
 			Name:      o.GetName(),
 		}, existingObj)
 
-		if existingObj == nil {
+		if errors.IsNotFound(err) {
 			// if not create one
-			err := integration.Client.Create(ctx, &o)
+			err = integration.Client.Create(ctx, &o)
 			s.Require().NoError(err)
 
-			s.T().Log("created: ", o.GroupVersionKind().String(),
+			s.T().Log("Created: ", o.GroupVersionKind().String(),
```

Comment:
I don't think we capitalize the Log string across our codebase. So do you mind reverting this :)

https://github.com/openshift/addon-operator/blob/6ca9401e355952374e29c22b1d6665b8fbbea0a1/integration/integration_test.go#L37

https://github.com/openshift/addon-operator/blob/6ca9401e355952374e29c22b1d6665b8fbbea0a1/integration/teardown_test.go#L37

https://github.com/openshift/addon-operator/blob/6ca9401e355952374e29c22b1d6665b8fbbea0a1/integration/resource_adoption_test.go#L32

https://github.com/openshift/addon-operator/blob/6ca9401e355952374e29c22b1d6665b8fbbea0a1/integration/helpers.go#L179

https://github.com/openshift/addon-operator/blob/6ca9401e355952374e29c22b1d6665b8fbbea0a1/integration/helpers.go#L183